src/Microsoft.OData.Edm/Microsoft.OData.Edm.csproj

```diff
@@ -94,9 +94,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="System.Text.Json">
-      <Version>4.7.1</Version>
-    </PackageReference>
+    <PackageReference Include="System.Text.Json" Version="4.*" />

```

Comment:
@xuzhg Curious how `Version=4.*` is interpreted. Wouldn't 4.7.1 be validated against 4.*?

Reply:
Here's the context:

Floating versions
A floating dependency version is specified with the * character. For example, 6.0.*. This version specification says "use the latest 6.0.x version"; 4.* means "use the latest 4.x version." Using a floating version reduces changes to the project file, while keeping up to date with the latest version of a dependency.

When using a floating version, NuGet resolves the highest version of a package that matches the version pattern, for example 6.0.* gets the highest version of a package that starts with 6.0:

Choosing version 6.0.1 when a floating version 6.0.* is requested

Reply:
@gathogojr Besides, I use this PR and output a nightly, That nightly can be used directly without the build error.

Reply:
@xuzhg Got it. So having the version pinned to 4.7.1 was the issue